tests/src/CLRTest.Execute.Bash.targets

```diff
@@ -255,8 +255,8 @@ $(BashCLRTestLaunchCmds)
 if [ ! -z ${RunCrossGen+x} ]%3B then
   TakeLock
 fi
-echo $_DebuggerFullPath $(_CLRTestRunFile) $ExePath $CLRTestExecutionArguments 
-$_DebuggerFullPath $(_CLRTestRunFile) $ExePath $CLRTestExecutionArguments 
+echo $_DebuggerFullPath $(_CLRTestRunFile) $ExePath %24(printf "'%s' " "${CLRTestExecutionArguments[@]}")
```

Comment:
I wonder what is the reason for using this obscure way of printing the arguments? Wouldn't just plain `${CLRTestExecutionArguments}` work?

Reply:
I think it doesn't work for strings with spaces and empty string:
For example,
```
echesako@echesako4:~$ CLRTestExecutionArguments=(" ")
echesako@echesako4:~$ echo "CLRTestExecutionArguments=${CLRTestExecutionArguments}"
CLRTestExecutionArguments=
echesako@echesako4:~$ echo CLRTestExecutionArguments=$(printf "'%s' " "${CLRTestExecutionArguments}")
CLRTestExecutionArguments=' '
echesako@echesako4:~$ CLRTestExecutionArguments=("")
echesako@echesako4:~$ echo "CLRTestExecutionArguments=${CLRTestExecutionArguments}"
CLRTestExecutionArguments=
echesako@echesako4:~$ echo CLRTestExecutionArguments=$(printf "'%s' " "${CLRTestExecutionArguments}")
CLRTestExecutionArguments=''
```

Reply:
Maybe there is a simpler way of doing this?

Reply:
My original intention was to echo an exact command that can be copy-pasted to repro the test run

Reply:
For example, `/opt/code/bin/tests/Linux.x64.Checked/Tests/Core_Root/corerun ThreadStartString.exe ''` as opposed to `/opt/code/bin/tests/Linux.x64.Checked/Tests/Core_Root/corerun ThreadStartString.exe`

Reply:
How about just 
```sh
echo "CLRTestExecutionArguments='${CLRTestExecutionArguments[@]}'"
```
Seems to work for both cases you've mentioned

Reply:
Btw, your way doesn't seem to work for the case of three arguments:
```sh
export CLRTestExecutionArguments=(a b c)
echo CLRTestExecutionArguments=$(printf "'%s' " "${CLRTestExecutionArguments}")

Output:
CLRTestExecutionArguments='a'
```

Reply:
Oh, nice! Thank you! I will simplify this part

Reply:
I made a typo in my example, the expression should be `echo CLRTestExecutionArguments=$(printf "'%s' " "${CLRTestExecutionArguments[@]}")`:
```
echesako@echesako4:~$ CLRTestExecutionArguments=(a b c)
echesako@echesako4:~$ echo CLRTestExecutionArguments=$(printf "'%s' " "${CLRTestExecutionArguments[@]}")
CLRTestExecutionArguments='a' 'b' 'c'
```